templates/focus-ios.xliff

```diff
@@ -269,6 +291,18 @@
         <source>Open Maps</source>
         <note>Button label in Maps URL dialog to open Maps. Test page: https://people-mozilla.org/~bnicholson/test/schemes.html</note>
       </trans-unit>
+      <trans-unit id="FindInPage.Done">
+        <source>Find in page done</source>
+        <note>Done button in Find in Page Toolbar.</note>
```

Comment:
I'm guessing this is also an Accessibility label (not 100% sure though). If so, would be good to call out in the localization note.

Reply:
If it's not, I find the string sounds a bit weird, because the note says it's a Done button - however that doesn't sound like a Done button from the string itself ("Find in page done")

Reply:
Yes, this is an accessibility label. I will add that as a note in the comment!